app/Http/Controllers/ProfileController.php

```diff
@@ -159,12 +160,14 @@ public function passwordSave(Request $request)
         });
 
         if (! $validator->fails()) {
-            $user->password = Hash::make($request->input('password'));
-            $user->save();
 
+            $user->password = Hash::make($request->input('password'));
+            // We have to use saveQuietly here because for some reason this method was calling the User Oserver twice :(
```

Comment:
'Observer'

Reply:
Also, *very* curious as to why this seemed to fire the Observer twice? Weird. But, well, this fixes it so I'm fine with it.